src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.h

```diff
@@ -131,6 +131,7 @@ class wholeBodyDynamicsDeviceFilters
  * | defaultContactFrames |      -   | vector of strings |  -    |    -          | Yes      | If not data is read from the skin, specify the location of the default contacts | For each submodel induced by the FT sensor, the first not used frame that belongs to that submodel is selected from the list. An error is raised if not suitable frame is found for a submodel. |
  * | useJointVelocity     |        - | bool              |  -    |      true     |  No      | Select if the measured joint velocities (read from the getEncoderSpeeds method) are used for estimation, or if they should be forced to 0.0 . | The default value of true is deprecated, and in the future the parameter will be required. |
  * | useJointAcceleration |        - | bool              |  -    |      true     |  No      | Select if the measured joint accelerations (read from the getEncoderAccelerations method) are used for estimation, or if they should be forced to 0.0 . | The default value of true is deprecated, and in the future the parameter will be required. |
+ * | streamFilteredFT |        - | bool              |  -    |      false     |  No      | Select if the filtered and offset removed forces will be streamed or not. | The value streamed by this ports is affected by the secondary calibration matrix, the estimated offset and temperature coefficients ( if any ). |
```

Comment:
I think we need to document also the name of the ports streamed.

Reply:
Is it enough to put the syntax rule or all of the names?

Reply:
The syntax rule.